src/renault_api/cli/renault_vehicle.py

```diff
@@ -203,16 +207,21 @@ async def update_battery_status(
         ):
             response.chargingStatus = 0.0
 
-        items = [
-            ("Battery level", response.batteryLevel, "%"),
-            ("Last updated", response.timestamp, "tzdatetime"),
-            ("Available energy", response.batteryAvailableEnergy, "kWh"),
-            ("Range estimate", response.batteryAutonomy, "km"),
-            ("Plug state", response.get_plug_status(), None),
-            ("Charging state", response.get_charging_status(), None),
-            ("Charge rate", response.chargingInstantaneousPower, "kW"),
-            ("Time remaining", response.chargingRemainingTime, "min"),
-        ]
+        if ctx_data["json"]:
+            items = [
+                ("battery-status", response.raw_data, None),
+            ]
```

Comment:
Try this: to have it as a structured dict:
```suggestion
            status_table["battery-status"] = response.raw_data
            return
```

Reply:
yes that's It ! I'm struggling formatting code in my github comment :)